### PR TITLE
Add entity framework

### PR DESCRIPTION
**Pull request by KazzModan**

### CODE REVIEW

File: .vs/Holy_Locket/FileContentIndex/read.lock
Comment (0lyaMelnyk) [inline .vs/Holy_Locket/FileContentIndex/read.lock:1]: .`vs` folder is under `gitignore`, don't push it to remote

File: Holy_locket.BLL/Holy_locket.BLL.csproj [hunk 1]
```diff
@@ -7,6 +7,16 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="7.0.13" />
```
Comment (0lyaMelnyk) [inline Holy_locket.BLL/Holy_locket.BLL.csproj:11]: BLL _shouldn't_ know details about DAL realization, you don't need to add these packages to BLL project

File: Holy_locket.WebAPI/Controllers/GetDoctorsController.cs
Comment (0lyaMelnyk) [inline Holy_locket.WebAPI/Controllers/GetDoctorsController.cs:1]: I believe these changes aren't related to your ticket and they will be merged from the separate pr

File: Holy_locket.WebAPI/Holy_locket.WebAPI.csproj [hunk 2]
```diff
@@ -9,8 +9,15 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="7.0.13" />
```
Comment (0lyaMelnyk) [inline Holy_locket.WebAPI/Holy_locket.WebAPI.csproj:12]: the same as a BLL project

**Review by 0lyaMelnyk — COMMENTED** [review]
Congrats on your first pr! Take a look at my comments and address them!

**Comment by KazzModan** [comment]
Edited

File: Holy_locket.BLL/Holy_locket.BLL.csproj
Comment (0lyaMelnyk) [inline Holy_locket.BLL/Holy_locket.BLL.csproj:1]: why do we need backup projects?

Comment (0lyaMelnyk) [reply Holy_locket.BLL/Holy_locket.BLL.csproj:1]: sorry, miss commenting, I mean backup.bll.csproj file

**Review by 0lyaMelnyk — APPROVED** [review]
Resolve conflicts and it looks good to merge
